tests/formats_test/selinux_test.py

```diff
@@ -33,10 +36,9 @@ def testMountingExt2FS(self):
         an_fs.unmount()
         os.rmdir(mountpoint)
 
-        self.assertEqual(root_selinux_context[1], 'system_u:object_r:file_t:s0')
+        self.assertEqual(root_selinux_context[1], DEFAULT_CONTEXT)
```

Comment:
It's useless. At some point I think it was expected that you'd get a different context while the thing was mounted, but only if `flags.installer_mode` was True. Now it appears that selinux doesn't care about this mountpoint at all. There are two clear options:

1. remove this test completely
2. rewrite the test to use mock to directly verify that we're calling the appropriate selinux function where appropriate (bearing in mind that python2-mock is an epel package)